test_integration/test_frequency_hint_usage.py

```diff
@@ -207,12 +206,13 @@ def test_frequency_sane_ee(self):
 
         # EE agent should assign not the minimum frequency for regions
         # with the COMPUTE hint.
-        self.assertNotEqual(self._freq_min, assigned_freq, msg='Expected assigned frequency to not be the min from policy since this region has the COMPUTE hint. ({} == {}).'.format(self._freq_min, assigned_freq))
-        util.assertNotNear(self, self._freq_min, achieved_freq, msg='Expected achieved frequency to NOT be near min from policy. ({} ~= {})'.format(self._freq_min, achieved_freq))
+        self.assertNotEqual(self._freq_min, assigned_freq,
+                            msg='Expected assigned frequency to not be the min from policy since this region has the COMPUTE hint. ({} == {}).'.format(self._freq_min, assigned_freq))
+        util.assertNotNear(self, self._freq_min, achieved_freq,
+                           msg='Expected achieved frequency to NOT be near min from policy. ({} ~= {})'.format(self._freq_min, achieved_freq))
 
 
 if __name__ == '__main__':
     # Call do_launch to clear non-pyunit command line option
     util.do_launch()
     unittest.main()
```

Comment:
This newline is vital. :P

I'm noticing a trend where patches I write introduce stuff like this, and patches that Chris does remove them...